lib/api/v3/notifications/property_factory.rb

```diff
@@ -53,11 +55,19 @@ def schemas_for(notifications)
       ::API::V3::Values::Schemas::ValueSchemaFactory.all_for(detail_properties)
     end
 
+    # Returns the outward facing notification group attributes
+    def groups_for(values)
```

Comment:
This is [unspeced on unit level|(https://github.com/opf/openproject/blob/dev/spec/lib/api/v3/notifications/property_factory_spec.rb).

I would also welcome a [request spec](https://github.com/opf/openproject/blob/dev/spec/requests/api/v3/notifications/index_resource_spec.rb#L239-L268) that ensures, that both types of date alerts (start/due date) are aggregated into the same group

Reply:
Done:
- https://github.com/opf/openproject/pull/11558/files#diff-16321f9a19c4815392963b1c1e8d3e231dbdacff98281814dc70c3d64ddaa916R171-R209
- https://github.com/opf/openproject/pull/11558/files#diff-6dda4d2b88f3ed6045d6415ab8af40404b4389d6fc49e05fd08402a47e0e3b55R283